brainrender_napari/widgets/atlas_manager_view.py

```diff
@@ -38,7 +45,11 @@ def __init__(self, parent: QWidget = None):
         """
         super().__init__(parent)
 
-        self.setModel(AtlasTableModel(AtlasManagerView))
+        self.table = AtlasTableModel(AtlasManagerView)
+        self.proxy = QSortFilterProxyModel()
```

Comment:
```suggestion
        self.source_model = AtlasTableModel(AtlasManagerView)
        self.proxy_model = QSortFilterProxyModel()
```
And rename accordingly? I think it's useful to be explicit.